components/pixels/lib.rs

```diff
@@ -632,7 +678,7 @@ fn decode_static_image(
         return None;
     };
     let mut rgba = dynamic_image.into_rgba8();
-    rgba8_byte_swap_colors_inplace(&mut rgba);
+    let is_opaque = rgba8_premultiply_inplace(&mut rgba);
```

Comment:
self-commentary: I though historically BGRA was more optimized and less problematic, but apparently not: https://github.com/servo/webrender/wiki/Swizzling-story